includes/User.php

```diff
@@ -3016,8 +2992,8 @@ public function clearAllNotifications() {
 			);
 
 			wfRunHooks( 'User::resetWatch', array ( $id ) );
-		# 	We also need to clear here the "you have new message" notification for the own user_talk page
-		#	This is cleared one page view later in Article::viewUpdates();
+			# 	We also need to clear here the "you have new message" notification for the own user_talk page
+			#	This is cleared one page view later in Article::viewUpdates();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)